conda/recipes/librmm/recipe.yaml

```diff
@@ -114,6 +115,7 @@ outputs:
         - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
         - ${{ pin_subpackage("librmm", exact=True) }}
         - rapids-logger =0.1
+        - cuda-compat-impl
```

Comment:
We should never need compat packages as a RAPIDS dependency. Why was this added?
```suggestion
```